packages/zwave-js/src/lib/controller/Controller.ts

```diff
@@ -123,7 +123,7 @@ interface ControllerEventCallbacks {
 	"heal network done": (result: ReadonlyMap<number, HealNodeStatus>) => void;
 }
 
-type ControllerEvents = Extract<keyof ControllerEventCallbacks, string>;
+export type ControllerEvents = Extract<keyof ControllerEventCallbacks, string>;
```

Comment:
@AlCalzone any reason you prefer `Extract<keyof ControllerEventCallbacks, string>` instead of `keyof ControllerEventCallbacks` ?

Reply:
I'm sure there was a reason for this 🤷🏻‍♂️